ui/handle_input.go

```diff
@@ -418,3 +452,31 @@ func (win *Window) sortTransactions() {
 		return frontTime.Before(backTime)
 	})
 }
+
+func (win *Window) updateToTransactionDetailsBtns() {
+	win.inputs.toTransactionDetails = make(map[string]*gesture.Click)
+	for _, walTxs := range win.walletTransactions.Txs {
+		for _, txn := range walTxs {
+			win.inputs.toTransactionDetails[txn.Txn.Hash] = &gesture.Click{}
+		}
+	}
+}
+
+func viewTxnOnBrowser(hash string) {
+	var err error
+	url := "https://testnet.dcrdata.org/tx/" + hash
```

Comment:
you can add a "getBlockExplorerUrl" method to the wallet package. The method should return the url with respect to the network the app is running on.